### PR TITLE
Use the higher-level sort printer for universe inconsistencies.

### PR DESCRIPTION
**Pull request by ppedrot**

### CODE REVIEW

File: checker/checker.ml [hunk 1]
```diff
@@ -242,10 +242,17 @@ let explain_exn = function
   | Sys.Break ->
     hov 0 (fnl () ++ str "User interrupt.")
   | UGraph.UniverseInconsistency i ->
+    let prs u = match u with
+    | Sorts.Set -> str "Set"
+    | Sorts.Prop -> str "Prop"
+    | Sorts.SProp -> str "SProp"
+    | Sorts.Type u -> Univ.Universe.pr_with Univ.Level.pr u
+    | Sorts.QSort _ -> assert false
```
Comment (Alizter) [inline checker/checker.ml:250]: You will print something here right?

**Comment by SkySkimmer** [comment]
The test suite rejects this change

**Comment by ppedrot** [comment]
Hmm, given the tests doesn't that mean we only ever pass sorts that are a single level?

**Comment by ppedrot** [comment]
So it seems we abuse the universe inconsistency exception from UGraph to mean several things. It should only talk about levels since this is the only thing UGraph knows about, but the upper layers reuse it to talk about sorts.

**Comment by SkySkimmer** [comment]
>So it seems we abuse the universe inconsistency exception from UGraph to mean several things. It should only talk about levels since this is the only thing UGraph knows about, but the upper layers reuse it to talk about sorts.

UGraph knows about sorts though. It has check_leq for sorts and so on.

**Comment by ppedrot** [comment]
I am closing this PR because I am now convinced we never want to print the quality there.

**Pull request closed without merging**